src/run.ts

```diff
@@ -30,6 +29,9 @@ async function run(helper: Helper) {
 
 		const fileQuestions = getFileQuestions(packageName, matchingCSSFiles, cssDataFileExtension);
 		const selectedWidgets = await askForDesiredFiles(fileQuestions);
+		if (!selectedWidgets.length) {
+			throw new Error('No widgets were selected');
```

Comment:
Should this be an error or simply a cli exit with a message to the user stating that they've failed to select any widgets to theme?

Reply:
Personally I think it's an error because the tool isn't being used in the expected way if no widgets are selected (e.g. it will have no effect), what do you think?

Reply:
What does it look like output wise? Can you post a screenshot please?

Reply:
It looks like this, the thing I kinda like it the semantics, e.g. you used the tool in an unexpected way (because there's not really a use case to select 0 widgets) and therefore the error code + error message style is that of an error!

Here's the screenshot:

![image](https://user-images.githubusercontent.com/381495/34682088-c89f876a-f495-11e7-8da5-99274d2667c2.png)

Reply:
ok that looks fine. thanks